protobuf-c.spec

```diff
@@ -1,19 +1,39 @@
 Name:           protobuf-c
-Version:        1.3.0
+Version:        1.3.1
 Release:        1%{?dist}
 Summary:        C bindings for Google's Protocol Buffers
 
-
+%if 0%{?suse_version} >= 1315
+Group:          Development/Libraries/C and C++
+License:        BSD-2-Clause
+%else
 Group:          System Environment/Libraries
 License:        BSD
+%endif
+
 URL:            https://github.com/protobuf-c/protobuf-c
 Source0:        https://github.com/protobuf-c/protobuf-c/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
+%if 0%{?suse_version} >= 1315
+BuildRequires:  pkg-config
+%endif
+
 %description
 Protocol Buffers are a way of encoding structured data in an efficient yet
 extensible format. This package provides a code generator and run-time
 libraries to use Protocol Buffers from pure C (not C++).
 
+%if 0%{?suse_version} >= 1315
+%package -n lib%{name}1
+Summary: C bindings for Google's Protocol Buffers
+License:        BSD-2-Clause
+%description -n lib%{name}1
+Protocol Buffers are a way of encoding structured data in an efficient yet
+extensible format. This package provides a code generator and run-time
+libraries to use Protocol Buffers from pure C (not C++).
+%endif
+
+%if 0%{?suse_version} == 0
```

Comment:
Why don't we want this subpackage on sles?

Reply:
SUSE lint is not happy about the package having no modules.

So should we be building it for CentOS?

Reply:
I'd think probably not.  Or are we misplacing a file that should be in this sub-package into another sub-package?  Here's [RH/Fedora's specfile](https://src.fedoraproject.org/rpms/protobuf-c/blob/master/f/protobuf-c.spec) for this.  How are we deviating?

Reply:
Your name is in the change log of this for removing the only file from this package.

Reply:
So, the simple test is to add the file back and see if the build complains that it's not there.

Ahhh.  https://src.fedoraproject.org/rpms/protobuf-c/blob/master/f/protobuf-c.spec#_42 vs. https://github.com/daos-stack/protobuf-c/pull/4/files/7fa52e245f4ed2c2e3ff0cae0a38e87d0e4f5681#diff-265c681b433f2d9a8288397ebabc80feR65

But I don't recall the reason for adding `--disable-protoc`.  Probably easiest is to just remove it and see what happens.

Reply:
There is a reference to a bug about parallel make builds failing.
Enabling it fails on the SUSE builds because it needs a compiler that supports C++ language features.
It fails on CentOS because the protobuf package was not found by configure.

Reply:
After doing all I can quickly:
1. el7 can not build protoc because the distro provided protobuf package is too old.
2. SUSE builds protoc once the BuildDepends are setup.